doc/source/topology_system.rst

```diff
@@ -0,0 +1,89 @@
+.. -*- coding: utf-8 -*-
+
+=====================
+The topology system
+=====================
+
+MDAnalysis groups static data about a :code:`Universe` into its topology. This is typically loaded from a topology file. Topology information falls into 3 categories:
+
+    * Atom containers (Residues and Segments)
+    * Atom attributes (e.g. name, mass, bfactor)
+    * Topology objects: bonds, angles, dihedrals, impropers
+
+Users will almost never interact directly with a :code:`Topology`. Modifying atom containers or topology attributes is typically done through :code:`Universe`. Methods for viewing containers or topology attributes, or for calculating topology object values, are accessed through :code:`AtomGroup`.
+
+Atom containers
+===============
+
+To add a :code:`Residue` or :code:`Segment` to a topology, use the :code:`Universe.add_Residue` or :code:`Universe.add_Segment` methods.
+
+.. code-block::
```

Comment:
The example is ok but better would be one of the FAQs: "How do I create different "chains" for protein consisting of multiple polypeptides?" (e.g., when loaded from a GRO file) or how do I create custom segments for "protein", "lipids", "solvent"?

Reply:
https://www.mdanalysis.org/docs/documentation_pages/topology/init.html#supported-topology-formats

Reply:
There's now a section in `constructing_universe.ipynb` that creates custom segments for protein domains. I could also make a mini-notebook of this, if that doesn't seem redundant.

Reply:
Also, moved it to the Groups of atoms page.